code/__DEFINES/components.dm

```diff
@@ -6,10 +6,10 @@
 
 // How multiple components of the exact same type are handled in the same datum
 
-#define COMPONENT_DUPE_HIGHLANDER 0		//old component is deleted (default)
-#define COMPONENT_DUPE_ALLOWED 1		//duplicates allowed
-#define COMPONENT_DUPE_UNIQUE 2			//new component is deleted
-#define COMPONENT_DUPE_UNIQUE_PASSARGS 4	//old component is given the initialization args of the new
+#define COMPONENT_DUPE_HIGHLANDER 		0		//old component is deleted (default)
+#define COMPONENT_DUPE_ALLOWED			(1<<0)	//duplicates allowed
+#define COMPONENT_DUPE_UNIQUE 			(1<<1)	//new component is deleted
+#define COMPONENT_DUPE_UNIQUE_PASSARGS 	(1<<2)	//old component is given the initialization args of the new

```

Comment:
These aren't bitflags, it's just a sequence which happens to skip 3.

Reply:
Why is it skipping 3?